_guides/2021-04-23-Dispelling-myths-qualifying-assumptions.md

```diff
@@ -93,6 +93,11 @@ Overall, OSS has made great improvements in the quality of code produced. These
 
 Proprietary software, on the other hand, may be more prone to bugs, but less susceptible to malicious intent. (Security issues are often discovered and patched faster in OSS.) ([5](https://merlcenter.org/guides/Dispelling-myths-qualifying-assumptions/#footnotes)) Many proprietary software applications also tend to be more user-friendly than their OSS alternatives, as design and function of the software are often developed simultaneously and with the user in mind. Resource constraints and ownership can cause the user experience of OSS to be deprioritized.
 
+<p align="center">
+  <img src="/assets/img/posts/myth-assumption.png" alt="Myth and assumption in OSS" width="100%">
+  <i>Summary of common myths and assumptions in OSS</i>
```

Comment:
What do you think about making "Summary of common myths and assumptions in OSS" a heading above the graphic instead of having it at the bottom?